modules/custom-tuning-specification.adoc

```diff
@@ -82,9 +121,41 @@ The individual items of the list:
 <7> Optional operand configuration.
 <8> Turn debugging on or off for the TuneD daemon. Options are `true` for on or `false` for off. The default is `false`.
 <9> Turn `reapply_sysctl` functionality on or off for the TuneD daemon. Options are `true` for on and `false` for off.
+endif::rosa-hcp-tuning[]
+ifdef::rosa-hcp-tuning[]
+[source,json]
+----
+{
+  "profile": [
+    {
+    # ...
+    }
+  ],
+  "recommend": [
+    {
+      "profile": <tuned_profile_name>, <1>
+      "priority": <priority>, <2>
+      "machineConfigLabels": { <Key_Pair_for_MachineConfig> <3>
+      },
+      "match": [ <4>
+        {
+          "label": <label_information> <5>
+        },
+      ]
+    },
+  ]
+}
+----
+<1> Profile ordering priority. Lower numbers mean higher priority (`0` is the highest priority).
+<1> A TuneD profile to apply on a match. For example `tuned_profile_1`.
+<1> Optional: A dictionary of key/value `MachineConfig` labels. The keys must be unique.
```

Comment:
This term is getting "called out" by vale. Sure enough, the style guide recommends against using "/" at all. I very much know what you mean but is there a different way to say it that's clearer?